src/store/ducks/generic/MDAPlans.ts

```diff
@@ -0,0 +1,106 @@
+import {
```

Comment:
This duck is very similar to the one for IRS and MDA-Point.  We can make this more DRY: https://github.com/onaio/reveal-frontend/issues/1143